File: darcula-themes/pygments/pygments_base16/base16-scheme.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#2B2B2B'
    base01 = '#323232'
    base02 = '#214283'
    base03 = '#808080'
    base04 = '#D0D0D0'
    base05 = '#D8D8D8'
    base06 = '#E8E8E8'
    base07 = '#F8F8F8'
    base08 = '#79ABFF'
    base09 = '#9876AA'
    base0a = '#A9B7C6'
    base0b = '#A5C25C'
    base0c = '#629755'
    base0d = '#FFC66D'
    base0e = '#CC7832'
    base0f = '#D25252'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('scheme', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords

File: darcula-themes/crosh/colors/base16-scheme.js
// Base16 Darcula
// Scheme: Matthias Brandt (github.com/PlusMinus0) based on Darcula theme by Konstantin Bulenkov (github.com/bulenkov/Darcula)

var color_scheme = {
        'base00': '#2B2B2B',
        'base01': '#323232',
        'base02': '#214283',
        'base03': '#808080',
        'base04': '#D0D0D0',
        'base05': '#D8D8D8',
        'base06': '#E8E8E8',
        'base07': '#F8F8F8',
        'base08': '#79ABFF',
        'base09': '#9876AA',
        'base0A': '#A9B7C6',
        'base0B': '#A5C25C',
        'base0C': '#629755',
        'base0D': '#FFC66D',
        'base0E': '#CC7832',
        'base0F': '#D25252',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(216, 216, 216, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);
